ledger/src/sigverify_shreds.rs

```diff
@@ -177,8 +181,10 @@ fn shred_gpu_offsets(
             let sig = shred::layout::get_signature_range();
             let sig = add_offset(sig, pubkeys_end);
             debug_assert_eq!(sig.end - sig.start, std::mem::size_of::<Signature>());
-            let msg = shred::layout::get_signed_message_range(packet);
-            let msg = add_offset(msg, pubkeys_end);
+            let shred = shred::layout::get_shred(packet);
+            // TODO: signature verification should fail for empty messages!
```

Comment:
Is there something to do for the "TODO"? Or just let the verification fail if the buffer is empty?

Reply:
changed the comment